packages/feedback-service/package.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "feedback",
+  "description": "Feedback micro-service for One Platform",
+  "opservice": {
+    "className": "Feedback",
+    "serviceName": "feedback"
+  },
+  "version": "",
+  "keywords": [
+    "web-components",
+    "html"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "github:1-Platform/one-platform",
+    "directory": "packages/feedback-service"
+  },
+  "publishConfig": {
+    "access": "private"
+  },
+  "private": true,
+  "files": [
+    "dist"
+  ],
+  "scripts": {
+    "test": "jest --forceExit",
+    "build:dev": "webpack --config webpack.dev.js",
+    "start:dev": "webpack --watch --config webpack.dev.js",
+    "build": "webpack --config webpack.prod.js",
+    "start": "webpack --watch --config webpack.prod.js"
+  },
+  "author": {
+    "name": "Anjnee K Sharma",
+    "email": "anjnee.k.sharma@gmail.com"
```

Comment:
@sayak-sarkar Shouldn't this be red hat email?

Reply:
Yup!